readthedocs/projects/forms.py

```diff
@@ -376,16 +376,19 @@ def clean_parent(self):
 
     def clean_child(self):
         child = self.cleaned_data['child']
-
-        child.is_valid_as_subproject(
-            self.project, forms.ValidationError
-        )
+        if self.instance.pk is None:
+            child.is_valid_as_subproject(
+                self.project, forms.ValidationError
+            )
```

Comment:
I'm not sure to understand the logic behind this. Can you add a comment in the code explaining what's the context for this check here?